src/vm/disassembler.cpp

```diff
@@ -78,16 +77,49 @@ void Disassembler::StaticInitialize()
 #if USE_COREDISTOOLS_DISASSEMBLER
     _ASSERTE(!IsAvailable());
 
-    // TODO: The 'coredistools' library will eventually be part of a NuGet package, need to be able to load
-    // that using appropriate search paths
-    LPCWSTR libraryName = MAKEDLLNAME(W("coredistools"));
-    HMODULE libraryHandle = CLRLoadLibrary(libraryName);
+	LPCWSTR libraryName = MAKEDLLNAME(W("coredistools"));
+  	// Try to load library
+	HMODULE libraryHandle = CLRLoadLibrary(libraryName);
+    
+#if !defined(_WIN32) 
```

Comment:
I think we should use explicit path to attempt/find the library regardless of platforms (even though Windows can find it in the same location of the current binary) to get consistent error/warning message.
I just searched the code base a little bit, and it appears you could use the following sequence without introduction of ifdef-out.
```
GetModuleFileNameW -- get module path
PathRemoveFileSpecW -- strip file name to get directory path only
PathCombineW -- combine the directory path with coredistools name.
```